src/EditorFeatures/Core/Copilot/RoslynProposalAdjusterProvider.cs

```diff
@@ -191,9 +191,10 @@ bool ReportFailureTelemetry(Exception ex)
             var (proposedEdits, formatGroup, adjustmentResults) = proposalAdjusterService is null
                 ? default
                 : await proposalAdjusterService.TryAdjustProposalAsync(
-                    document, CopilotEditorUtilities.TryGetNormalizedTextChanges(editGroup), cancellationToken).ConfigureAwait(false);
+                    allowableAdjustments: null, document,
+                    CopilotEditorUtilities.TryGetNormalizedTextChanges(editGroup), cancellationToken).ConfigureAwait(false);
 
-            if (proposedEdits.IsDefault)
+            if (proposedEdits.IsDefault || adjustmentResults.IsDefaultOrEmpty)
```

Comment:
i'm find the usages of default non-default a bit confusing.  have we doc'ed as to when you might get default vs empty?